### PR TITLE
Car Wash

### PR DESCRIPTION
**Pull request by Serhii-Hrynyshyn**

### CODE REVIEW

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]
Fix flake8, please. If you need help with it, you can write to chat or attend a Q&A

File: .flake8 [hunk 1]
```diff
@@ -1,7 +1 @@
...
-max-complexity = 18
-select = B,C,E,F,W,T4,B9,ANN,Q0,N8,VNE
-exclude = venv, tests
+[flake8]
inline-quotes = "
ignore = E203, E266, W503, ANN002, ANN003, ANN101, ANN102, ANN401, N807, N818
max-line-length = 79
max-complexity = 18
select = B,C,E,F,W,T4,B9,ANN,Q0,N8,VNE
exclude = venv, tests
```
Comment (olena-hul) [inline .flake8:1]: I think you should close the PR, and reopen it with the new solution. Make sure flake8 and tests are passing

Comment (olena-hul) [reply .flake8:1]: If you need help, feel free to go to the Q&A

**Review by olena-hul — CHANGES_REQUESTED** [review]

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]
The file looks broken. Can you close this PR and create a new one? If you need help you can attend the Q&A

**Pull request closed without merging**